unikraft/component/component.go

```diff
@@ -59,7 +60,14 @@ func TranslateFromSchema(props interface{}) (map[string]interface{}, error) {
 		for key, prop := range entry {
 			switch key {
 			case "version":
-				component["version"] = prop.(string)
+				switch tprop := prop.(type) {
+				case string:
+					component["version"] = tprop
+				case int:
+					component["version"] = strconv.Itoa(tprop)
+				default:
+					component["version"] = fmt.Sprintf("%s", prop)
```

Comment:
Is the format string really required here?

```suggestion
					component["version"] = fmt.Sprint(prop)
```